packages/protocol/contracts/rrp/requesters/interfaces/IRrpBeaconServer.sol

```diff
@@ -42,4 +53,9 @@ interface IRrpBeaconServer is IMetaAdminnable, IWhitelister {
         external
         view
         returns (int224 value, uint32 timestamp);
+
+    function sponsorToUpdateRequesterToPermissonStatus(
```

Comment:
Just curious but is it standard practice to explicitly state public mappings/structs in the interface? Shouldn't you be able to query them regardless? I've seen this done in the TemplateUtils as well.